core/cloudflow-operator/src/main/scala/cloudflow/operator/Operator.scala

```diff
@@ -77,14 +77,52 @@ object Operator {
     )
   }
 
+  def handleConfigurationInput(
+      client: KubernetesClient
+  )(implicit system: ActorSystem, mat: Materializer, ec: ExecutionContext, ctx: DeploymentContext) = {
+    val logAttributes  = Attributes.logLevels(onElement = Attributes.LogLevels.Info)
+    val actionExecutor = new SkuberActionExecutor()
+    // only watch secrets that contain input config
+    val watchOptions = ListOptions(
+      labelSelector = Some(
+        LabelSelector(
+          LabelSelector.IsEqualRequirement(CloudflowLabels.ManagedBy, CloudflowLabels.ManagedByCloudflow),
+          LabelSelector.IsEqualRequirement(CloudflowLabels.ConfigFormat, CloudflowLabels.InputConfig)
+        )
+      ),
+      resourceVersion = None
+    )
+    runStream(
+      watch[Secret](client, watchOptions)
```

Comment:
I would add some comments here what the chain of events are when a config change is detected.

Reply:
Also, if this fails, is this failure reflected in any status section?

Reply:
Good point, that's something I can add!

Reply:
Oh, on the other hand, if a pod fails, the status is updated anyway, which could be caused (hypothetically) by a bad config, though the config is validated by the cli.

Reply:
Since we don't have any cause-effect between config change and pod failing, I think this is good enough for now.